src/app/Settings/FeatureLevels.tsx

```diff
@@ -0,0 +1,95 @@
+ * person obtaining a copy of this software, associated documentation and/or data
+ * (collectively the "Software"), free of charge and under any and all copyright
+ * rights in the Software, and any and all patent rights owned or freely
+ * licensable by each licensor hereunder covering either (i) the unmodified
+ * Software as contributed to or provided by such licensor, or (ii) the Larger
+ * Works (as defined below), to deal in both
+ *
+ * (a) the Software, and
+ * (b) any piece of software and/or hardware listed in the lrgrwrks.txt file if
+ * one is included with the Software (each a "Larger Work" to which the Software
+ * is contributed by such licensors),
+ *
+ * without restriction, including without limitation the rights to copy, create
+ * derivative works of, display, perform, and distribute the Software and make,
+ * use, sell, offer for sale, import, export, have made, and have sold the
+ * Software and the Larger Work(s), and to sublicense the foregoing rights on
+ * either these or other terms.
+ *
+ * This license is subject to the following condition:
+ * The above copyright notice and either this complete permission notice or at
+ * a minimum a reference to the UPL must be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import * as React from 'react';
+import { ServiceContext } from '@app/Shared/Services/Services';
+import { useSubscriptions } from '@app/utils/useSubscriptions';
+import { UserSetting } from './Settings';
+import { FeatureLevel } from '@app/Shared/Services/Settings.service';
+import { Select, SelectOption } from '@patternfly/react-core';
+
+const Component = () => {
+  const context = React.useContext(ServiceContext);
+  const addSubscription = useSubscriptions();
+  const [state, setState] = React.useState(0);
+  const [open, setOpen] = React.useState(false);
+
+  React.useLayoutEffect(() => {
+    addSubscription(context.settings.featureLevel().subscribe((level) => setState(level)));
+  }, [addSubscription, context.settings.featureLevel, setState]);
+
+  const handleToggle = React.useCallback(() => {
+    setOpen((v) => !v);
+  }, [setOpen]);
+
+  const handleSelect = React.useCallback(
+    (_, v) => {
+      setState(v);
+      context.settings.setFeatureLevel(v);
+      setOpen(false);
+    },
+    [setState, setOpen, context.settings.setFeatureLevel]
+  );
+
+  return (
+    <>
+      <Select isOpen={open} onToggle={handleToggle} selections={FeatureLevel[state]} onSelect={handleSelect}>
```

Comment:
![feture-level](https://user-images.githubusercontent.com/68053619/208169768-ae4d441b-ea02-4797-b262-86204950d7ef.png)

Maybe we should use `<Select  isFlipEnabled menuAppendTo={'parent'} />` to enable flip here?

Reply:
![flip-enabled](https://user-images.githubusercontent.com/68053619/208170162-2635a6b4-664a-40b8-934b-1c4c4f288945.png)